<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shaker</class>
 <widget class="QWidget" name="Shaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shaker</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>采集卡 AO参数设置</string>
   </property>
   <widget class="QDoubleSpinBox" name="ao_fre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>频率(0~1000)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>振幅(-1V~1V)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ao_amp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>采样率</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ao_sample">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="ao_dev">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="text">
     <string>ao0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道名称</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="recv_time">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>144</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>144</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>采集时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>191</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件缩放</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_file_zoom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>186</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;</string>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>186</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> V</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>采集卡 AI参数设置</string>
   </property>
   <widget class="QSpinBox" name="acc_sample">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>采样率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Hz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="acc_dev">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="text">
     <string>ai0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道名称</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="acc_en">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加速度计采集使能</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Hz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="force_en">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>力传感器采集使能</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="force_dev">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="text">
     <string>ai1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>采样率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道名称</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="force_sample">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QCustomPlot" name="GraphRT" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>191</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>采集卡</string>
   </property>
   <widget class="QLineEdit" name="dev_name">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
    </property>
    <property name="text">
     <string>Dev1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设备名称:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>波形显示</string>
    </property>
    <widget class="QRadioButton" name="zoom_v">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>纵向伸缩</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="zoom_h">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>横向伸缩</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="zoom_g">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>全局伸缩</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="sine_btn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { /* all types of tool button */
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
    padding-right: 20px; /* make way for the popup button */
}

QToolButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
QToolButton::menu-button {
    border: 2px solid gray;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
    /* 16px width + 4px for border = 20px allocated above */
    width: 16px;
}

QToolButton::menu-arrow {
    image: url(downarrow.png);
}

QToolButton::menu-arrow:open {
    top: 1px; left: 1px; /* shift it a bit */
}</string>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sine_let">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string>选择波形文件路径</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>播放选择</string>
   </property>
   <widget class="QRadioButton" name="radioButton_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件波形</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正弦波形</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 7.5px;
    background-color: rgb(91, 155, 213);
	font: 75 10pt &quot;微软雅黑&quot;;
	color: white;
}

QPushButton:hover{
	background-color: rgb(0, 176, 240);
	color: white;
}

QPushButton:pressed{
	background-color:rgb(92, 163, 99);                                                           	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>激振器输出反馈</string>
   </property>
   <widget class="QPushButton" name="start_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 7.5px;
    background-color: rgb(91, 155, 213);
	font: 75 10pt &quot;微软雅黑&quot;;
	color: white;
}

QPushButton:hover{
	background-color: rgb(0, 176, 240);
	color: white;
}

QPushButton:pressed{
	background-color:rgb(92, 163, 99);                                                           	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
